modules/serverless-creating-subscriptions-ODC.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_workflows/serverless-channels.adoc
+
+[id="serverless-creating-subscriptions-ODC_{context}"]
+= Connecting a channel to a sink using subscriptions in {product-title} web console
+
+You can create a subscription to connect a service or other event sink to a channel through the {product-title} web console.
+
+.Prerequisites
+* To create a subscription through the web console, you must already have created a Knative service that can be used as an event sink.
+
+.Procedure
+. Navigate to the *Topology* page in the web console *Developer* perspective.
+. Create a subscription using one of the following methods:
+.. Click on the channel that you want to create a subscription for, and drag the arrow. The option will appear to *Add Subscription*.
++
+image::create-sub-ODC.png[Create a subscription for the channel]
++
+... Select the *Service* you want to create a subscription for from the drop-down list.
+... Click *Add*.
+.. Right-click on the channel that you want to add a subscription for, and select *Add Subscription* from the context menu.
```

Comment:
Suggestion: Drop this method. These are the 2 ways the user can complete the procedure. IMHO, we should recommend the best way to do it.